### PR TITLE
Clean up: setup

### PR DESCRIPTION
**Pull request by UlrikeS91**
MINOR changes:
- improved instructions
- establish alphabetical order
- fixed indentation
- renamed `setupType` to `type` to reduce list of property names and because we decided to use type when the specific context is explicit

MAJOR changes:
none

SPECIAL ATTENTION:
none

### CODE REVIEW

**Review by Peyman-N — APPROVED** [review]

File: schemas/products/setup.schema.tpl.json [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
+    "components",    
+    "description",
+    "name"
+  ],
+  "properties": {    
+    "components": {
+      "type": "array",
+      "minItems": 2,
+      "uniqueItems": true,
+      "_instruction": "Add all components, including other setups, that are part in this setup. The setup must be composed of at least one device.",
```
Comment (lzehl) [inline schemas/products/setup.schema.tpl.json]: this seems like contradiction... minItems 2 but only one device is required according to the instructions...

**Review by lzehl — CHANGES_REQUESTED** [review]

File: schemas/products/setup.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/products/setup.schema.tpl.json]: I suggest to just delete the second sentence.

**Review by lzehl — CHANGES_REQUESTED** [review]

**Review by lzehl — APPROVED** [review]
updated instructions of "components" to "Add all components, including other setups, that are part in this setup. Note that a setup should not be only composed of software."

File: schemas/products/setup.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/products/setup.schema.tpl.json]: changed to "Add all components, including other setups, that are part in this setup. Note that a setup should not be only composed of software."

**Pull request merged**